act-rules-format.bs

```diff
@@ -126,30 +122,33 @@ Each [=accessibility requirement=] in the mapping MUST include the following:
 3. a link to the [=accessibility requirements document=] if one exists, and
```

Comment:
change "link" to "a link or reference" (eg. if the document is not publicly available or directly linkable for any other reason)